books/RayTracingTheNextWeek.html

```diff
@@ -1965,13 +2037,34 @@
 
 <div class='together'>
 The representation of a packed array in that order is pretty standard. Thankfully, the [stb_image][]
-package makes that super simple -- just include the header `rtw_stb_image.h` (found in the project
-source code at `src/common/rtw_stb_image.h`) in `main.h`:
+package makes that super simple -- just write a header called `rtw_stb_image.h` that also deals with
+some compiler warnings:
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
-    #include "rtw_stb_image.h"
+    #ifndef RTWEEKEND_STB_IMAGE_H
+    #define RTWEEKEND_STB_IMAGE_H
+
+    // Disable pedantic warnings for this external library.
+    #ifdef _MSC_VER
+        // Microsoft Visual C++ Compiler
+        #pragma warning (push, 0)
+    #endif
+
+    #define STB_IMAGE_IMPLEMENTATION
+    #include "external/stb_image.h"
+
+    // Restore warning levels.
+    #ifdef _MSC_VER
+        // Microsoft Visual C++ Compiler
+        #pragma warning (pop)
+    #endif
+
+    #endif
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [incl-stb-img]: Including the STB image package]
+    [Listing [rtw-stb-image]: <kbd>[rtw_stb_image.h]</kbd> Include the stb_image package]
+
+The above assumes that you have copied the `stb_image.h` into a folder called `export`. Adjust as per
```

Comment:
“a folder called ~export~ `external`.”

Also, a suggestion: “Adjust ~as per~ according to your directory structure.”